stan/math/prim/scal.hpp

```diff
@@ -59,6 +59,8 @@
 #include <stan/math/prim/scal/err/invalid_argument_vec.hpp>
 #include <stan/math/prim/scal/err/out_of_range.hpp>
 
+#include <stan/math/prim/scal/err/is_size_match.hpp>
```

Comment:
alphabetize this include. It should go after `invalid_argument_vec.hpp`. And remove the surrounding whitespace.